index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="vewport" content="width=device-width, initial-scale=1.0">
+    <title>Documents</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+ <boby>
+    <nav class="navbar">
+        <div class="name">Patrick & Amos</div>
+        <div class="links">
+            <a href="#about">About</a>
+            <a href="#project">Project</a>
+            <a href="#contact">Contact</a>
+        </div>
+    </nav>
+
+    <div class="main-section">
+        <section id="about">
+            <header class="header">
+                <h1>About</h1>
+                <h2>Patrick & Amos</h2>
+                <h2>Web Developer</h2>
+            </header>
+            <div class="aboutme">
+                <p>
+                    we are skilled web developer that enjoys being practical in our projects
+                </p>     
+            </div>
+                <section id="projects">
+                    <header class="header">
+                        <h1>Projects</h1>
+                        <h2>Here are some of the projects we made</h2>
+                    </header>
+                    <div class="project">
+                        <div class="project-title">Official Logo</div>
+                        <img src="images/IMG-20210410-WA0001 (1).jpg" alt="survey" class="project-img">
+                    </div>
+                    <div class="project">
+                        <div class="project-title">Logo Design</div>
+                        <img src="images/IMG-20210410-WA0002.jpg" alt="survey" class="project-img">
+                    </div> 
+        </section>
+        <section id="contact">
+            <h1>Contact</h1>
+            <div class="main-contacts">
+                <div class="contact-type">
+                    <h3>Email</h3>
+                    <P>
+                        amosmaina0877@gmail.com
+                    </P>
+                </div>
+                <div class="contact-type">
+                    <h3>Phone</h3>
+                    <P>
+                        +25456855187
+                    </P>
+                </div>
+                <div class="contact-type">
+                    <h3>Email</h3>
+                    <P>
+                        patrickkeita80@yahoo.com
+                    </P>
+                </div>
+                <div class="contact-type">
+                    <h3>Phone</h3>
+                    <P>
+                        +24453895096
+                    </P>
+                </div>
+            </div>
+        </section>
+    </div>
+ </boby>
```

Comment:
I think in this you meant to use <body> instead of the <boby>

Reply:
Kindly make sure that your suggestions are clear and concise. Also please don't forget to explain why you requested a change. it might be obvious for you but not for someone else. :+1: :smiley: :smiley: